utils/src/com/cloud/utils/StringUtils.java

```diff
@@ -32,6 +33,21 @@
 public class StringUtils {
     private static final char[] hexChar = {'0', '1', '2', '3', '4', '5', '6', '7', '8', '9', 'A', 'B', 'C', 'D', 'E', 'F'};
 
+    private static Charset _ACS_Charset;
+
+    {
+        String preferredCharset = "UTF8";
```

Comment:
Actually it would work, UTF8 is a supported alias for the UTF-8 charset. I do prefer to always use the actual register charset name "UTF-8" though ;)